app/upgrades.go

```diff
@@ -92,6 +93,7 @@ var upgrades = map[string]appUpgrade{
 
 			return vm, nil
 		},
+		Added: []string{triggertypes.ModuleName},
```

Comment:
This is needed in the `"rust-rc1"` upgrade too.

Reply:
Thanks for finding this.